scripts/abi_artifact_generator/templates/describe_functions.inc.jinja_template

```diff
@@ -65,7 +65,9 @@ void describe_/*( name )*/(void) {
       /*( fill_stack_with_random_data )*/
       /*( fill_registers_with_random_data )*/
       /*( call_a_function(name) )*/
+#ifndef CALLEE_CLEANED_ARGUMENTS
       /*( restore_stack )*/
```

Comment:
A more appropriate name would be `cleanup_stack_arguments`

Reply:
Ah, my bad! I was under the assumption that it is used elsewhere, but no, it looks like I removed the other usage at some point because it wasn't necessary.

Reply:
What do you mean?
I was suggesting to rename `cleanup_stack_arguments` into  `restore_stack`?

Reply:
Renamed it.